versions.json

```diff
@@ -222,6 +222,14 @@
         "arm64v8",
         "ppc64le",
         "s390x"
+      ],
+	  "alpine3.13": [
```

Comment:
Yeah, I think that should be it, just spacing
```suggestion
     "alpine3.13": [
```

Reply:
This is fixed :)